go.mod

```diff
@@ -2,52 +2,77 @@ module github.com/hashicorp/consul-dataplane
 
 go 1.20
 
-// This replace directive is needed because `api` requires 0.4.1 of proto-public but we need an unreleased version
-replace github.com/hashicorp/consul/proto-public v0.4.1 => github.com/hashicorp/consul/proto-public v0.1.2-0.20230929231147-632fd65c091c
-
 require (
 	dario.cat/mergo v1.0.0
 	github.com/armon/go-metrics v0.4.1
 	github.com/hashi-derek/grpc-proxy v0.0.0-20231207191910-191266484d75
 	github.com/hashicorp/consul-server-connection-manager v0.1.6
-	github.com/hashicorp/consul/proto-public v0.4.1
-	github.com/hashicorp/go-hclog v1.2.2
+	github.com/hashicorp/consul/proto-public v0.1.2-0.20240122214449-ea12838e3dc1
```

Comment:
TODO: replace with actual `consul/proto-public` version after release of: https://github.com/hashicorp/consul/pull/20257